### DISCUSSION TITLE
Does ava support mock-import?

### DISCUSSION THREAD

**Discussion by johnsonsamuel** [avajs/ava#3069] [Q&A]
Hi,

Just wanted to check if ava supports mock-import

https://github.com/coderaiser/mock-import

**Comment by novemberborn** [comment]
AVA doesn't do anything special on top of Node.js CJS and ESM systems, so I reckon that'd be fine.

**Reply by johnsonsamuel** [reply]
Thanks for taking time to reply @novemberborn :). I did a test today to verify `ava` with `mock-import` and the mocking does not seem to be working but the same works with `tap`.
